src/auth/bearer/fake/fake.cc

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2014, Treehouse Networks Ltd. New Zealand
```

Comment:
That was not an oversight, the copyright in question is part of the BSD license.
 "
 * 1. Redistributions of source code must retain the above copyright
 * notice, this list of conditions and the following disclaimer.
"

This is not newly minted code. It was written years ago for Treehouse and customers - and not exclusively by me BTW (IIRC I just cropped out the unnecessary business logic parts from a real helper and set the user="fake" output line, was done with the basic auth fake helper around the same timeframe). Which is why I said "the author" in my earlier comment, and also why its not my name there in the statement, and the year is not 2017.

My options here under the BSD License are to go in with the license + copyright as it was supplied with. Or to drop the helper entirely and hope something else gets written to fill the gap.

As for our policy. IIRC that applies to code written specifically for inclusion in Squid, and the Squid boilerplate disclaimers. All the helpers and code that were written independently from Squid retain whatever their license and copyrights were under that umbrella Squid boilerplate.

For example; if you were to write a library (lets say paxxer) and place a specific GPL-compatible license and your own copyright statement on it, then either of us merged it wholesale into Squid that copyright would remain. The changes needed to make it fit into Squid and any modifications done afterwards would fall under the policy and not get additional copyright statements from the modifiers.

My expectation is that would apply to this particular helper code too. As mentioned above, I cropped out some business logic from the original code for this helper - but did not add my own statement to the file in the process since that work was specifically for Squid.